docs/Lifecycle/index.md

```diff
@@ -0,0 +1,257 @@
+# **SDL Development Lifecycle**
+You are welcome to participate in the evolution of SDL. Because of high quality expectation of SDL community, every contribution has to pass certain processing before it becomes an SDL evolutionary step.<br>
+To make introduction of new ideas into SDL transparent and manageable, we outline in this document major stages of development lifecycle.<br>
+We would like to emphasize that every contribution has to conform to agreed upon [definition of done (DoD)][DOD-LINK].
+
+## **Scope**
+SDL is a large product with multiple repositories. Some of them contains source codes for features and tests others mainly dedicated to documentation and some R&D experiments. The ones which do not strictly dedicated to documentation or R&D form set of product parts like SDL Core, ATF, ATF Scripts, HMI etc. Later in documentation those repositories are mentioned in generic manner: "product parts".
+
+## **Lifecycle processes**
+SDL lifecycle is based on three major processes:
+- [SDL Evolution][SDL-EP-LINK]
+- [SDL Contribution][SDL-CONT-LINK]
+- [SDL Release][SDL-REL-LINK]
+
+Next picture emphasize processes and their outcome in SDL.<br>
+![Lifecycle processes](assets/lifecycle.png "Lifecycle processes")<br>
+
+Each one of them might be executed in parallel or sequentially. However for each proposal there is a streamlined development process, outlined on following picture.<br>
+![Development streamline](assets/development-streamline.png "Development streamline")<br>
+
+## **Roles**
+SDL lifecycle considers interactions of following groups:
+
+- Stakeholders (primary responsible role SDL Project Manager)
+- [Change Control Board (CCB)][CCB-LINK] (primary responsible role CCB Group Manager)
+- Maintainers (primary responsible role is **Task Assignee**)
+- Community (primary responsible role for propsal review is **Proposal Author**)
+
+### Stakeholders
```

Comment:
I would like to keep this term so far. We do have different stakeholders: budget owners, PM, technical specialists, probably some community experts. All of them have concerns and might influence on project decisions. Final responsibility take SDL PM (which is not the Product Owner as such), he will share responsibility of classical Product Owner with CCB.